Documentation/Signoffs/AcceleratorSystem.md

```diff
@@ -0,0 +1,42 @@
+Accelerator Pedal Position System
+--------------------------------------
+### Function of the Subsystem
+The accelerator pedal position system will take the OEM signal from the pedal by splicing into the signal wires and input it to the local microcontroller. The local microcontroller will then calculate the position based the change of the voltage from the OEM hall effect sensor.
+
+### Specifications & Constraints
```

Comment:
This seems like a good first submission. The main issue is the constraints. They need to be considered more thoroughly and in more detail. Without a clear view of the constraints we can't actually analyze the system.